Directory.Build.props

```diff
@@ -6,6 +6,7 @@
       tell Microsoft.Common.props not to import Directory.Build.props again
     -->
     <ImportDirectoryBuildProps>false</ImportDirectoryBuildProps>
+    <NetCoreAppCurrent>netcoreapp5.0</NetCoreAppCurrent>
```

Comment:
You are duplicating this property which already exists in L33: https://github.com/dotnet/runtime/pull/457/files#diff-4b2f402501d23abbede6eac0f47783e4R33.